scripts/stubtest/allowlist.txt

```diff
@@ -186,3 +187,132 @@ django.urls.resolvers.URLPattern.lookup_str
 django.urls.resolvers.URLResolver.url_patterns
 django.urls.resolvers.URLResolver.urlconf_module
 django.utils.connection.BaseConnectionHandler.settings
+django.contrib.admin.helpers.InlineAdminFormSet.non_form_errors
```

Comment:
I've added them here without reordering to avoid a merge conflict with https://github.com/typeddjango/django-stubs/pull/1761